keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/Build.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.keel.api.artifacts
+
+/**
+ * An immutable data class that represents a published software artifact in the Spinnaker ecosystem.
+ *
+ * This class mirrors [com.netflix.spinnaker.igor.build.model.GenericBuild], but without all the Jackson baggage.
+ */
+
+data class Build(
```

Comment:
This could probably go in `keel-igor`...

Reply:
done